app/assets/stylesheets/govuk-component/_tasklist.scss

```diff
@@ -0,0 +1,358 @@
+.pub-c-tasklist {
```

Comment:
Is it possible to write this in a way that shows what's copied from an accordion, and what's specific to a tasklist?

Reply:
At the moment I fear it'll be difficult to make parts of this reusable, and too easy for the two patterns to grow out of sync.

Reply:
I think the tasklist has already deviated significantly from the accordion that keeping them in sync is neither possible or necessarily desirable. In fact, we're currently looking at a design proposal that would make the tasklist look very different. Happy to discuss though as I feel like I'm unaware of the merits involved.

Reply:
Hmm. I'm also worried that these two look very similar at the moment. At the very least, I'd hope that we could base both components on a "parent" component that encapsulates as much common behaviour as possible (eg expandable sections), and then make it as configurable as we need to meet the needs of both accordions and task lists. I think this is doable if we do more work with blocks, etc.

Reply:
That would be a good solution. Just to let you know I'm now looking at further changes to how the GA stuff works for task lists, which will further change the code involved.

Reply:
If tracking turns out to be a pain to commonise, a possible solution is to have no tracking by default, and people will have to implement their own tracking on a per-use basis (which could be done through JS).

Reply:
That would be great. I think I'd have done that from the start if the tracking on the accordion hadn't been so detailed and complex.